packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasSummary.ts

```diff
@@ -116,6 +133,18 @@ export function useGasSummary(): UseGasSummaryResult {
       return
     }
 
+    // If user has inputed an amount over their balance, don't estimate gas
```

Comment:
NIT: comment is off, this handles case when balance didn't load yet

Reply:
oops it got to here during merge conflict, i'll remove thanks